docs/general-reference/help-button.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: default
+title: Firebolt Help
+description: Describes the options of the Firebolt "Help" button
+nav_exclude: true
+search_exclude: false
+---
+
+# Firebolt Help Button
+{:.no_toc}
+
+This topic describes the options of the Firebolt "Help" button.
+
+## Getting to the Help Button
+Firebolt Help button is found in the bottom right corner of Firebolt's screen, appearing as a ```?``` sign.
+
+![Help Menu](../assets/images/getting_to_help.png)
+
+## Menu Options
+
+The menu has four different options. 
+
+
+![Help Menu](../assets/images/help_menu.png)
+
+
+### Status Page
+The status page allows you to view in real-time the operational status of our systems and descriptions of historical incidents. 
+
+You can also access the status page by browsing directly to it through https://firebolt.statuspage.io
+
+#### Main Page
```

Comment:
intentional different formatting than status page? If not, I think you want ### not ####

Reply:
The main page is a part of the status page:
-- Status page
     | -- Main page 
     | -- Historical uptime

So the levels were intentional.